moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -71,6 +74,8 @@ struct TrajectoryPointInfo {
 };
 typedef std::vector<TrajectoryPointInfo> TrajectoryInfo;
 
+typedef control_msgs::FollowJointTrajectoryGoal JointTrajectoryType;
```

Comment:
This should be a moveit_simple type and not a ROS type (as we discussed).

Reply:
done